apps/ima/behaviours/enquiries-email-local.js

```diff
@@ -0,0 +1,9 @@
+const config = require('../../../config');
+
+module.exports = superclass => class extends superclass {
+  locals(req, res) {
+    const locals = super.locals(req, res);
+    locals.queriesEmail = config.queriesEmail;
+    return locals;
+  }
+};
```

Comment:
If there is the potential to reuse this file to embed other emails in the future, maybe a more generic filename would suit e.g. set-confirmation-emails/ set-static-emails (just spitballing here)? If you do change the filename make sure to change the variable name in the test as well for consistency